### ISSUE TITLE
jQuery colorbox doesn't get focus in full-size view (element type "image")

### ISSUE THREAD

**Issue by pepesale** [contao/contao#7552]
### Affected version(s)

4.13, 5.3 and 5.4

### Description

When using the content element "image" and launching the full-size view the full size colorbox doesn't get the focus. This might be an issue concerning acessibility as the close-button cannot be focused with the tab key.

Checked this in the Contao demo by inserting a single image. In the content element "gallery" instead it works fine and the focus jumps inside the colorbox.

Checked in Contao 4.13, 5.3 and 5.4.

Checked this on Jack Moore's Colorbox Demonstration: Here it seems to work fine with colorboxes from single elements.

**Comment by fritzmg** [comment]
I cannot reproduce the issue. Reproduction I tried in Contao 4.13:

* Enabled __Include jQuery__ in the page layout.
* Enabled the `j_colorbox` template.
* Created a content element of type `image`.
* Selected an image and enabled __Full-size view/new window__.
* Opened the front end.
* Clicked on the image to open the full size image in the Colorbox.
* Pressed _Tab_ once.

The close button was selected.

I also see no reason why this could somehow be caused by Contao or its default `j_colorbox` template.

**Comment by pepesale** [comment]
That's what I exactly did in the official Contao demo. Tested with Firefox and Chrome.
Furthermore this was an issue which had been remarked from some external auditors for accessibility in some Contao websites (those without any further extensions).

**Comment by pepesale** [comment]
Tested right now again in official demo. The focus walks through all elements (including page footer) until it reaches finally the colorbox. But it should jump there directly when launching the colorbox, I think.

**Comment by fritzmg** [comment]
Yes, I can reproduce it, if other focusable elements come after the image.

But again: this is not related to Contao itself. It looks like the `trapFocus` feature of Colorbox is not working correctly or as intended. You'd have to report that [here](https://github.com/jackmoore/colorbox) - but Colorbox hasn't received a new release since 2016.
